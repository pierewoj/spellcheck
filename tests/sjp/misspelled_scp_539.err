1,25 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
3,17 Euclid: 
5,21 Przechowawcze: przechowawce
5,36 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
5,293 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
7,66 matteeriału: 
7,98 kroarędzi: 
7,548 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
11,33 pollcono: polecono
11,230 ubroarniowego: 
11,331 ubroarń: 
13,24 ubroarny: 
13,35 jaskroare: 
13,68 ubroarnie: 
15,33 pollcono: polecono
15,122 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
15,313 Carkharhinus: 
15,326 leucas: lucas
15,392 pawedać: 
15,485 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
15,531 woreto: loreto
17,235 sproardzić: 
17,444 Varanus: 
17,452 komodoensis: 
17,465 worean: 
17,697 proardopodobnie: 
19,162 proardopodobne: 
21,238 obroarz: 
21,479 obroarzu: 
21,503 matteeriał: 
