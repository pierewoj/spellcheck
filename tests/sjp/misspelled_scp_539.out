Identyfikator podmiotu: #SCP-539

Klasa podmiotu: #Euclid

Specjalne Czynności #Przechowawcze: #SCP-539 musi być przechowywany w wyściełanej skrzyni, która zawsze pozostaje zamknięta, o ile nieprzeprowadzane są testy zjawiska pod nadzorem pracownika z upoważnieniem co najmniej poziomu 2. Dopóki nie zakończą się odpowiednie badania, kontakt fizyczny z #SCP-539 jest wzbroniony.

Opis: Obiekt to cienki, okrągły dysk, wykonany z nierozpoznanego #matteeriału. Podczas gdy wzdłuż #kroarędzi obiektu można zidentyfikować rdzę, testy określiły, że ta nie jest produktem oksydacji podmiotu. Po rzuceniu, w najbliższej okolicy zajdzie zdarzenie, które natychmiast odwróci uwagę od rzucającego. Owe odwrócenia uwagi nie podlegają żadnemu obserwowalnemu wzorcowi. Jednakże potwierdzono, iż odwrócenie uwagi jest wprost proporcjonalnej intensywności do trudności w odwróceniu uwagi od rzucającego. W związku z nieprzewidywalnością efektu #SCP-539, testerom zaleca się wyrzucać obiekt jak najdalej od siebie.

Dodatek: Notatki z testów

Test 539-1: Pracownikom ochrony #pollcono stanąć w linii, aby mogli bez przeszkód obserwować testera, pod żadnym pozorem nie zrywając z nim kontaktu wzrokowego. Po rzuceniu dysku, pracownicy ochrony doświadczyli "nagłego problemu #ubroarniowego", przez co wszyscy zerwali kontakt wzrokowy z testerem, będąc bardziej przejęci utratą #ubroarń i kwestią otoczenia wywołującego niepokój u nagiego strażnika.

Test 539-2: Tester był #ubroarny w #jaskroare i zwracające nań uwagę #ubroarnie. Tester przebywał sam wraz z pracownikiem ochrony w pustym biurze. Pracownikowi ochrony rozkazano utrzymywać nieprzerwany kontakt wzrokowy z testerem. Po rzuceniu dysku, uruchomiony został alarm przeciwpożarowy budynku, a jego wyłączenie nie było możliwe aż do momentu odzyskania dysku i zresetowania komputerowego systemu ochrony.

Test 539-3: Pracownikom klasy D #pollcono "zwrócić na siebie uwagę lokalnych służb porządkowych", a następnie wykorzystać #SCP-539. Testerzy byli ścigani przez policję po rozbiciu szyby radiowozu. Po rzuceniu obiektu, na drodze ścigających testerów samochodów policyjnych pojawił się wielki przedstawiciel gatunku #Carkharhinus #leucas (Żarłacz tępogłowy). Policja zmuszona była się zatrzymać i #pawedać zjawisko dochodzeniu. Po odzyskaniu pracowników klasy D, zarówno rekin, jak i obiekt #SCP zostały odzyskane przez agentów polowych. #woreto odnotować, że funkcjonariusze policji, którzy doświadczyli zjawiska, nigdy nie wspomnieli o nim przełożonym.

Test 539-4: Test ten nie został autoryzowany i w związku z tym podjęto procedury dyscyplinarne o zróżnicowanym zakresie wobec zaangażowanych w jego przeprowadzenie osób. W ramach "dowcipu", grupa pracowników naukowych zdecydowała się #sproardzić, czy są w stanie odwrócić uwagę dr. ██████, znanego z poważnego zaangażowania w swoją pracę. Po rzuceniu obiektu, z szybu wentylacyjnego laboratorium wyszło stado dziesięciu przedstawicieli gatunku #Varanus #komodoensis (#worean z Komodo), a następnie przystąpiło do wypędzenia naukowca z pomieszczenia. Zwierzęta poruszały się niekontrolowane po całej placówce, dopóki nie podniesiono dysku — po czym to niemożliwe stało się ich zlokalizowanie, jako że #proardopodobnie opuściły placówkę. Ten "test" może wskazywać, że wykorzystanie zwierząt jest przeważającą metodą odwracania uwagi przez obiekt.

Słuchajcie. Lubię się pośmiać, jak każdy, ale to mogłoby się skończyć o wiele lepiej, gdyby pracownicy naukowi najpierw uzyskali zgodę. Ponadto uważam za bardzo #proardopodobne, że ta metoda odwrócenia uwagi będzie się wiązała ze zwierzętami, jako że te niemal zawsze zwracają na siebie uwagę ludzi — dr ██████.

Test 539-5: Test mający określić efektywność obiektu w stosunku do obserwatorów niebezpośrednich. Pracownicy testujący połączyli kamerę obserwacyjną z monitorem komputera w innym budynku. Komputer miał zarówno nagrywać, jak i wyświetlać #obroarz wideo. Kamerę skierowano w stronę testera, a pozostałe osoby przemieściły się przed monitor komputera. Po rzuceniu dysku, komputer doznał błędu krytycznego w związku z usunięciem sterowników sprzętowych, potrzebnych do kontrolowania #obroarzu wideo. Nagrany #matteeriał był zepsuty i niemożliwe było jego odzyskanie.
